web/modules/custom/dpl_login/src/Controller/DplLoginController.php

```diff
@@ -67,7 +62,7 @@ public function __construct(
    *
    * @return static
    */
-  public static function create(ContainerInterface $container) {
+  public static function create(ContainerInterface $container): static {
```

Comment:
Here we should be able to drop the `@return static` in the doc block.

We should consider [enforcing return type hints as close to the language level as possible in our coding standards](https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintsreturntypehint-).